tsparser/src/parser/resources/apis/encoding.rs

```diff
@@ -118,6 +121,9 @@ impl RequestEncoding {
                 ParamData::Query { .. } => by_loc.query.push(p),
                 ParamData::Body => by_loc.body.push(p),
                 ParamData::Cookie => by_loc.cookie.push(p),
+                ParamData::HTTPStatus => {
+                    // HTTPStatus can only be set in responses
```

Comment:
How does this behave if you try to use it in a request?

Reply:
its now handled like any other field if the encoding doesnt support http statuses (so it will use the default location)